indico/modules/events/abstracts/templates/abstract/review.html

```diff
@@ -1,9 +1,19 @@
+{% from 'events/abstracts/abstract/_common.html' import render_instruction_box %}
 {% from 'forms/_form.html' import form_header, form_rows, form_footer %}
 
 
 {% macro render_review_form(form, abstract=none, track=none, review=none, management=false) %}
     {% set url = url_for('.review_abstract', abstract, track, management=management) if not review else
                  url_for('.edit_review', review, management=management) %}
+
+    {% set event = abstract.event_new %}
+
+    {% call(link, endlink) render_instruction_box(event.cfa.reviewing_instructions, _("Reviewing instructions")) %}
+        {% trans -%}
+            Please don't forget to read the {{ link }}reviewing instructions{{ endlink }} before submitting a review.
```

Comment:
In the case of judgment you said "...before taking **your** decision", but in this case "...before submitting **a** review.
Shouldn't these 2 be the same? :thinking: 
Oh, also below "...before submitting **an** abtract"